lms/static/js/components/card/views/card.js

```diff
@@ -61,43 +62,47 @@
-                                 this.callIfFunction(this.cardClass);
+                        this.switchOnConfiguration('square-card', 'list-card') + ' ' +
+                        this.callIfFunction(this.cardClass);
                     if (this.callIfFunction(this.pennant)) {
                         result += ' has-pennant';
                     }
                     return result;
                 },
 
-                render: function () {
+                render: function() {
                     var maxLength = 72,
-                        description = this.callIfFunction(this.description);
+                        description = this.callIfFunction(this.description),
+                        $details;
                     if (description.length > maxLength) {
-                        description = description.substring(0, maxLength).trim() + '...'
+                        description = description.substring(0, maxLength).trim() + '...';
                     }
-                    this.$el.html(this.template({
-                        pennant: this.callIfFunction(this.pennant),
-                        title: this.callIfFunction(this.title),
-                        description: description,
-                        action_class: this.callIfFunction(this.actionClass),
-                        action_url: this.callIfFunction(this.actionUrl),
-                        action_content: this.callIfFunction(this.actionContent),
-                        configuration: this.callIfFunction(this.configuration),
-                        srInfo: this.srInfo
-                    }));
-                    var detailsEl = this.$el.find('.card-meta');
-                    _.each(this.callIfFunction(this.details), function (detail) {
+                    HtmlUtils.setHtml(
+                        this.$el,
+                        this.template({
+                            pennant: this.callIfFunction(this.pennant),
+                            title: this.callIfFunction(this.title),
+                            description: description,
+                            action_class: this.callIfFunction(this.actionClass),
+                            action_url: this.callIfFunction(this.actionUrl),
+                            action_content_html: this.callIfFunction(this.actionContentHtml),
+                            configuration: this.callIfFunction(this.configuration),
+                            srInfo: this.srInfo
+                        })
+                    );
+                    $details = this.$('.card-meta');
+                    _.each(this.callIfFunction(this.details), function(detail) {
                         // Call setElement to rebind event handlers
                         detail.setElement(detail.el).render();
                         detail.$el.addClass('meta-detail');
-                        detailsEl.append(detail.el);
+                        $details.append(detail.el);
```

Comment:
It isn't a problem because this is appending one DOM node to another. However, it may be a problem that our linter can't distinguish the two. Our new wrapper can't handle this case, at least as yet, so we can't convert it to anything else.